settings.gradle

```diff
@@ -76,6 +76,15 @@ List projects = [
   'qa:wildfly'
 ]
 
+File examplePluginsDir = new File(rootProject.projectDir, 'plugins/examples')
```

Comment:
And it is *different* from every other project because the path doesn't match the name, which I'd prefer not to do just to keep things consistent.